frontend/src/scenes/events/VolumeTable.scss

```diff
@@ -0,0 +1,5 @@
+.definition-description {
```

Comment:
Worth scoping styles for this component (e.g. under a `.volume-table` class). Also worth making a note in `EventsVolumeTable.tsx` to rename it to `VolumeTable.tsx` to match.

Reply:
Will put that in the todo for the next PR since I'll be adding more frontend pieces